app/models/push_policy.rb

```diff
@@ -0,0 +1,9 @@
+PushPolicy = Gem::Security::Policy.new(
```

Comment:
I chose not to define PushPolicy under the Gem::Security namespace because nothing else has that scope.  If rubygems winds up defining a Gem::Security::PushPolicy, then we can remove this one and switch to theirs.